### PR TITLE
search field changed into while loop

### PR DESCRIPTION
**Pull request by r4vanan**
i don't want kill that process while for searching the anime so i made a while loop for that.

### CODE REVIEW

**Comment by 71zenith** [comment]
appease the linter

**Comment by r4vanan** [comment]
> appease the linter

I can't understand what you mean sir

**Comment by r4vanan** [comment]
I think now it's ok

**Comment by RaynardGerraldo** [comment]
searching for anime as argument doesnt work with this, ./ani-cli naruto just prompts me to search anime again

File: ani-cli [hunk 1]
```diff
@@ -352,14 +359,12 @@ history)
...
-			printf "Search anime: " && read -r query
+			search_loop
 		else
 			query=$(: | external_menu "" "Search anime:")
 		fi
 	done
-	query=$(printf "%s" "$query" | sed "s| |%20|g")
-	anime_list=$(search_anime "$query")
-	[ -z "$anime_list" ] && die "No results found!"
+	search_loop
```
Comment (port19x) [inline ani-cli:367]: this would break external menu compatibility

**Review by port19x — CHANGES_REQUESTED** [review]

**Comment by r4vanan** [comment]
i think now `./ani-cli naruto` it can possible

**Comment by r4vanan** [comment]
now it can work with rofi

**Comment by RaynardGerraldo** [comment]
> i don't want kill that process while for searching the anime so i made a while loop for that.

can you explain what exactly this means and what it improves? because I dont see a difference, you can show some screenshots

**Comment by r4vanan** [comment]
if i search the anime name. if the result is not found means it kill the process i dont like that process. so i changed that and do the loop thing. like if you enter the wrong or invalid name it will ask the name again. it will work in rofi too. no matter how many time you entered wrong name. if the result not found means it will ask the name again

**Comment by RaynardGerraldo** [comment]
> if i search the anime name. if the result is not found means it kill the process i dont like that process. so i changed that and do the loop thing. like if you enter the wrong or invalid name it will ask the name again. it will work in rofi too. no matter how many time you entered wrong name. if the result not found means it will ask the name again

ok, got it

**Comment by port19x** [comment]
This is straightup poorly implemented. Not sure if you can refactor it, or if it needs to be this branchy in order to work

**Comment by r4vanan** [comment]
You can close it. Because I like this thing so I modified like this

**Pull request closed without merging**